clustertask/github-update-cd-repo/templates/clustertask.yaml

```diff
@@ -73,31 +73,45 @@ spec:
               name: cd-repo-ssh-token
               key: token
       args:
-        - -c
-        - |
+        - '-c'
+        - >
           set -e
           # Get the cd repo to clone
           # Find the protocol
           PROTOCOL=$(echo $params.CD_REPO_URL| cut -d @ -f1 | cut -d : -f1)
-          
-          string = $(params.CD_REPO_URL)
+
+          string=$(params.CD_REPO_URL)
           REPO_NAME_WITH_ORG=${string#*$".com"} 
           REPO_NAME_WITH_ORG=${REPO_NAME_WITH_ORG:1}
-          
           GIT_ORGANIZATION=$(echo $REPO_NAME_WITH_ORG | cut -d'/' -f1)
           CD_REPO_NAME=$(echo $REPO_NAME_WITH_ORG | cut -d'/' -f2)
-          
-          if [ "$(PROTOCOL)" == "https" ]; then
-          git clone https://${GIT_USERNAME}:${GIT_PASSWORD}@github.com/$(GIT_ORGANIZATION)/$(CD_REPO_NAME)
+
+          if [ $PROTOCOL == "https" ]; then
+          git clone
+          https://${GIT_USERNAME}:${GIT_PASSWORD}@github.com/$(GIT_ORGANIZATION)/$(CD_REPO_NAME)
           git config --global user.password ${GIT_PASSWORD}
           git config --global user.name ${GIT_USERNAME}
           git config --global user.email ${GIT_EMAIL}
           else
-          echo "$cd_repo_ssh_token" >> "~/.ssh/id_rsa"
-          git clone $params.CD_REPO_URL
+          git config --global user.name tekton-bot
+          git config --global user.email asfa@stakater.com
```

Comment:
@AsfaMumtaz hardcoding your email?

Reply:
Oops. Was just testing it out. This needs to have the stakater email.